### PR TITLE
add new file

### PR DESCRIPTION
**Pull request by devbymo**
I'm just adding new file with small text content

### CODE REVIEW

**Comment by devbymo** [comment]
check it please

**Comment by motest11** [comment]
make more changes please

File: new-file.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+by devbymo (me)
```
Comment (motest11) [inline new-file.txt]: make it more unserstandable

Comment (motest11) [inline new-file.txt]: add comments

**Review by motest11 — APPROVED** [review]

**Review by motest11 — CHANGES_REQUESTED** [review]
review it again

**Pull request merged**